### ISSUE TITLE
XCTest/XCTest.h not found

### ISSUE THREAD

**Issue by pisacode** [olofd/react-native-photos-framework#93]

**Comment by pisacode** [comment]
React Native version 0.52 
xcode 10 

I am getting this error 

`/Users/pisacode/Desktop/appName/node_modules/react-native-photos-framework/ios/RNPhotosFrameworkTests/PHAssetsService_getAssetsForFetchResultTests.m:3:9: 'XCTest/XCTest.h' file not found`

Adding `$(PLATFORM_DIR)/Developer/Library/Frameworks` to `FRAMEWORK_SEARCH_PATHS` also did not work.

Is this package maintaining?

**Comment by ajaykumar97** [comment]
I have also got the same error. If anyone got the solution, please help.

**Comment by ajaykumar97** [comment]
Got the solution. Refer to this link
[https://stackoverflow.com/questions/24275470/xctest-xctest-h-not-found-on-old-projects-built-in-xcode-6/24651704#24651704](https://stackoverflow.com/questions/24275470/xctest-xctest-h-not-found-on-old-projects-built-in-xcode-6/24651704#24651704)

**Comment by chipqcl** [comment]
> Got the solution. Refer to this link
Hi @ajaykumar97 ,

There are about 10 different suggested solutions in that stackoverflow thread (including the FRAMEWORK_SEARCH_PATHS one listed above).  Is there a specific fix that worked for you?

Thanks!

**Comment by ajaykumar97** [comment]
Yes! I found another workaround for this, because this error comes again after few builds. The workaround was to delete Tests folder of react-native-photos-framework from the  pod development folder(as I am using pod to use this framework).

**Comment by MobileAppVault** [comment]
Works for me as well! Many thanks @ajaykumar97

**Comment by rintoj** [comment]
Delete two items from compile sources:

![screenshot 2019-02-13 at 4 41 40 am](https://user-images.githubusercontent.com/10824903/52674666-4e583600-2f4a-11e9-844a-0f85929b0d3f.png)
